inlanecruising_plugin/src/inlanecruising_plugin.cpp

```diff
@@ -115,14 +115,14 @@ void InLaneCruisingPlugin::plan_trajectory_callback(
     if (yield_client_ && yield_client_->service_is_ready())
     {
       RCLCPP_DEBUG_STREAM(nh_->get_logger(), "Yield Client is valid");
-      
+
       auto yield_srv = std::make_shared<carma_planning_msgs::srv::PlanTrajectory::Request>();
       yield_srv->initial_trajectory_plan = original_trajectory;
       yield_srv->vehicle_state = req->vehicle_state;
 
       auto yield_resp = yield_client_->async_send_request(yield_srv);
 
-      auto future_status = yield_resp.wait_for(std::chrono::milliseconds(100));
+      auto future_status = yield_resp.wait_for(std::chrono::milliseconds(300));
```

Comment:
I am going to make this a parameter. This should be same parameter as the plan_delegator's timeout. Furthermore, other tactical plugins may need to call this yield functionality so this parameter needs to be accessible universally.

Reply:
I was thinking I can add this file to vehicle calibration file maybe (or VehicleConfigParams in carma-config)? please let me know

Reply:
I think VehicleConfigParams in carma-config is a good place.

Reply:
added to carma-config:
TRB specific: https://github.com/usdot-fhwa-stol/carma-config/pull/290
develop changes: https://github.com/usdot-fhwa-stol/carma-config/pull/289